packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -465,7 +465,9 @@ test('agoricName invitation source errors', async t => {
     }),
     {
       message:
-        '{"source":"agoricContract","instancePath":["psm-IST-AUSD"]} - Must have missing properties ["callPipe"]',
+        // TODO The pattern is here only as a temporary measure to tolerate
+        // the property order being sorted and not.
+        /\{("instancePath":\["psm-IST-AUSD"\]|,|"source":"agoricContract")*\} - Must have missing properties \["callPipe"\]/,
```

Comment:
```suggestion
        /\{("instancePath":\["psm-IST-AUSD"\]|,|"source":"agoricContract"){3}\} - Must have missing properties \["callPipe"\]/,
```

or perhaps just use `+`

or skip all that and only require matching `Must have ...`.

Reply:
Love it! Done.